_posts/2024-08-09-contributing-to-eclipse-che-docs.adoc

```diff
@@ -0,0 +1,102 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
+categories: []
+keywords: ['che-docs', 'documentation', 'write', 'contribute']
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+== The quick and easy way to write Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che cloud development environment (CDE).
+
+image::/assets/img/contributing-to-eclipse-che-docs/che-docs.png[An Eclipse Che CDE created from the che-docs repository.]
+
+
+== Getting started
+
+Click the Developer Workspace button in the che-docs repository README file.
+This opens a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the link:https://developers.redhat.com/developer-sandbox[Developer Sandbox].
+Alternatively, you can click link:https://workspaces.openshift.com#https://github.com/eclipse/che-docs[*right here*].
+
+== How does it work?
+
+What about the che-docs repository makes using an Eclipse Che CDE to contribute content so convenient? The following two elements:
+
+* link:https://eclipse.dev/che/docs/stable/end-user-guide/devfile-introduction/[devfile]
+* list of link:https://eclipse.dev/che/docs/stable/end-user-guide/microsoft-visual-studio-code-open-source-ide/#automating-installation-of-microsoft-visual-studio-code-extensions-at-workspace-startup[automatically installed Microsoft Visual Studio Code extensions]
+
+Let's look at the devfile first.
+The che-docs repository contains a simple devfile
+that defines the basic features of the che-docs workspace.
+It also includes a command for building a preview of your newly created content:
+
+[source, code]
+----
+# sh tools/preview.sh
```

Comment:
for some reason executing the command does not work for me 
<img width="1521" alt="Screenshot 2024-08-07 at 19 40 14" src="https://github.com/user-attachments/assets/38549f69-8142-41ef-a310-26431a2ce5b0">

Reply:
@dkwon17 @deerskindoll does it work for you?

Reply:
oh no. same result. I tried to run the script locally and there's an error message: `tools/preview.sh: line 15: gulp: command not found`

Reply:
locally you might not have gulp ;-) It should work on Developer Sandbox OOTB because all dependencies are part of the image - that's exactly the problem we are solving - contributing with zero install

Reply:
my point is that it worked locally without issues. I'll look into it.

Reply:
once fixed I believe we need to update the first screenshot with the workspace & preview image

Reply:
it works again. please check if it works for you, too.

Reply:
works like a charm \o/ 
please, replace the first screenshot with:
<img width="1727" alt="Screenshot 2024-08-09 at 11 37 51" src="https://github.com/user-attachments/assets/8e363c05-620d-4786-89c4-bffdfa017b95">